src/main/java/seedu/address/logic/commands/AssignTaskCommand.java

```diff
@@ -0,0 +1,120 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_EMPLOYEE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PROJECT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
+
+import java.util.List;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.employee.Employee;
+import seedu.address.model.employee.UniqueEmployeeList;
+import seedu.address.model.project.Project;
+import seedu.address.model.task.Task;
+import seedu.address.model.task.TaskList;
+
+/**
+ * Assigns a task in a project to employee(s) in that project.
+ */
+public class AssignTaskCommand extends Command {
+    public static final String COMMAND_WORD = "assignT";
+    public static final String MESSAGE_SUCCESS = "Assigned task: %1$s to employee, %2$s";
+    public static final String MESSAGE_USAGE =
+            COMMAND_WORD + ": Assigns a task in a project to an employee in the project.\n"
+            + "Parameters: "
+            + PREFIX_PROJECT + "PROJECT_INDEX "
+            + PREFIX_TAG + "TASK_INDEX "
```

Comment:
I think it would be better to create and use PREFIX_TASK instead of PREFIX_TAG. It does not matter that both of them share the same prefix "/t" currently, and if we decide to change one of the prefix, the other will still remain the same.

Reply:
Ah yes, I was already changing them to task but I must have missed these, thanks!